_topics/en/time-warp.md

```diff
@@ -93,13 +93,13 @@ Bitcoins DAA only adjusts difficulty once every 2016 blocks.  Miners who
 begin a time warp attack cannot significantly affect difficulty for the
 first 2016-block epoch because both the first and last blocks in that
 epoch will need to have approximately accurate times.  For the second
-epoch, miners can make it appear that the first block was mined two
+epoch, miners can make it appear that the first block was mined four
 weeks in the past and the last block was mined at present, giving an
 average of about 20 minutes per block and causing the DAA to halve the
 difficulty.  Miners will then by able to complete the next epoch twice
 as fast and yet make it seem like it took about 25 minutes per block,
 lowering difficulty further.  They can repeat the attack indefinitely
-until they're producing one block per second, the lower bound allowed by
+until they're producing [six blocks per second]({{bse}}123698), the lower bound allowed by
```

Comment:
I dont think bse is is a global reference. Should probably just reference https://bitcoin.stackexchange.com/questions/123698/how-many-blocks-per-second-can-sustainably-be-created-using-a-time-warp-attack in the foot of the doc like a normal link

Reply:
Thanks, I amended the link.